frontend/src/lib/components/DateFilter/DateFilterRangeExperiment.scss

```diff
@@ -9,3 +9,11 @@
     border: 0 !important;
     font-size: 0.625em;
 }
+
+.ant-picker-range-arrow {
+    border-top: 1px solid var(--primary);
+    border-right: 1px solid var(--primary);
+}
+.ant-picker-range-wrapper {
+    border: 1px solid var(--primary);
+}
```

Comment:
These styles will apply to all such classes everywhere, not just this date picker. This may have some effect outside of just this component, I'm not sure.

Reply:
This is basically adding a blue border to the antd date calendar popup, which makes it style-coherent with our Lemon popups.

The next step would be to remove that component and create a LemonCalendarPicker, which I'm going to do after.

Is there a better place where to move these classes?

Reply:
Just sanity checking:
Will it show when the experiment is off? Do we use some other date pickers elsewhere in the app?

Reply:
Only in the DatePicker because the class is imported locally.